docs/sdks/remove-api-key.md

```diff
@@ -32,10 +32,23 @@ client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                         email='nemo@nautilus.com', password='xxxxx',
                         geofabric='_system')
 
-# Remove api key
-remove = client.remove_api_key('id1')
+# Check wether the chosen_api_key exists or not, comparing it to the list elements
+for apiElement in apiKeyIds:
+    if apiElement == CHOSEN_API_KEY_ID:
+        API_ACTIVE = True
+        break
 
-print(remove)
+# Checks if the API already exists and acts accordingly
```

Comment:
```suggestion
# Checks the specified API key
```

If I understand correctly, this step merely checks the API key, and the next one removes it. Correct?